samza-yarn/src/main/java/org/apache/samza/config/YarnConfig.java

```diff
@@ -190,4 +210,30 @@ public String getYarnCredentialsFile() {
   public String getYarnJobStagingDirectory() {
     return get(YARN_JOB_STAGING_DIRECTORY, null);
   }
+
+  public String getYarnApplicationViewAcl() {
```

Comment:
Minor: can inline these one-line methods / make them private since they're only used inside `getApplicationAcls`

Reply:
This file seems to have a getter for every config even though a couple of them are not used anywhere, so I'm simply following the pattern. More importantly, as I said, the view and modify acl are independently used and should be treated as two different configs. The Map<ApplicationAccessType, String> is tightly coupled with the setApplicationAcls method. I'm simply providing a helper function here